File: python/Answers/answer010.py
"""
Problem 10:
The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.

Find the sum of all the primes below two million.
"""

from python.utils import fast_primes


def answer():
    limit = int(2e6)
    return sum(int(p) for p in fast_primes(limit))


if __name__ == '__main__':
    print("Answer is:", answer())

File: julia/src/Answers/answer010.jl
#=
answer010:
- Julia version: 1.4.2
- Author: Zac
- Date: 2020-07-05

Problem 10:
The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.

Find the sum of all the primes below two million.

=#

include("../utils.jl")


answer(l=2000000) = sum(Utils.sieves(l))

# Output
println("Answer is: ", answer())
